src/components/breadcrumb-item/daikin-breadcrumb-item.ts

```diff
@@ -57,99 +20,79 @@ type LinkVariantProps = MergeVariantProps<typeof cvaLink>;
  *
  * ```html
  * <!-- See `daikin-breadcrumb` component for complete example. -->
- * <daikin-breadcrumb-item href="#">
- *   Breadcrumb Item 1
+ * <daikin-breadcrumb-item href="https://www.example.com">
+ *   Breadcrumb item
  * </daikin-breadcrumb-item>
  * ```
  */
 @customElement("daikin-breadcrumb-item")
 export class DaikinBreadcrumbItem extends LitElement {
   static override styles = css`
     ${unsafeCSS(tailwindStyles)}
-
-    :host {
-      display: inline-flex;
-      align-items: center;
-      gap: 0.5rem;
-      flex-shrink: 0;
-    }
-
-    :host([hidden]) {
-      display: none;
-    }
   `;
 
   /**
-   * Specify link href
+   * Specify link href.
    */
   @property({ type: String, reflect: true })
-  href = "";
+  href: string | null = null;
 
   /**
    * Specifies the display content.
    * If `ellipsis`, the "..." will be displayed instead of the link.
    * Set automatically by `daikin-breadcrumb`.
    */
   @property({ type: String, reflect: true })
-  variant: LinkVariantProps["variant"] = "normal";
+  variant: "normal" | "current" = "normal";
 
   /**
-   * Specify whether the link should be disabled
-   */
-  @property({ type: Boolean, reflect: true })
-  disabled = false;
-
-  /**
-   * Specify the link target
+   * Specify the link target.
    */
   @property({ type: String, reflect: true })
   target: string | null = null;
 
   /**
-   * Whether the slash after the link should shown.
+   * Whether or not to change the color of visited links.
    * Set automatically by `daikin-breadcrumb`.
    */
-  @property({ type: Boolean, reflect: true, attribute: "trailing-slash" })
-  trailingSlash = false;
+  @property({ type: Boolean, reflect: true, attribute: "show-visited" })
+  showVisited = false;
 
   /**
-   * Whether the item is the last one.
-   * When set to `true`, the item will also be considered to be the current page and the link will be disabled.
-   * Automatically set by `daikin-breadcrumb` component.
+   * _Internal use._
+   * Whether or not to display the divider on the right.
+   * Set automatically by `daikin-breadcrumb`.
+   *
+   * @private
    */
-  @property({ type: Boolean, reflect: true })
-  last = false;
+  @property({ type: Boolean, reflect: true, attribute: "show-divider" })
+  showDivider = false;
```

Comment:
nit. I'd like to make the position of the divider clear.

```suggestion
  @property({ type: Boolean, reflect: true, attribute: "append-divider" })
  appendDivider = false;
```